demo/node.go

```diff
@@ -121,6 +122,8 @@ func (n *Node) Start(certFolder string) {
 	var args = []string{"start"}
 	args = append(args, pair("--folder", n.base)...)
 	args = append(args, pair("--control", n.ctrl)...)
+	_, p, _ := net.SplitHostPort(n.addr)
+	args = append(args, pair("--listen", "0.0.0.0:"+p)...)
```

Comment:
let's switch to `127.0.0.1` to avoid exposing the demo openly without necesity !